INSTALL.md

```diff
@@ -26,133 +28,114 @@ hardware) by running `qvm-start sys-usb`.
 5. Update dom0 and install `make`, referring again to
 [the next section of SDW docs](https://workstation.securedrop.org/en/stable/admin/install.html#apply-dom0-updates-estimated-wait-time-15-30-minutes)
 
+In our case, we also install `open-vm-tools` and run `sudo systemctl enable vmtoolsd`,
+as our scripts use vmtoolsd to issue commands to the dom0 from the VMware API.
+
 ```
-sudo qubes-dom0-update make
+sudo qubes-dom0-update make open-vm-tools
 ```
 
 6. Run any updates you see in the Qubes menu and then reboot.
 
-7. In dom0, create the sd-ssh StandaloneVM:
+7. In dom0, create the sd-dev StandaloneVM. If it's Qubes 4.2, you can use the fedora-38-xfce template.
```

Comment:
Just FYI that F38 will be EOL in May - https://www.qubes-os.org/news/2024/02/13/fedora-39-templates-available-fedora-38-approaching-eol/

Based on the steps below I think it would be non trivial to switch, but in the long-run basing on Debian might be less maintenance because of the slower release cycle.

Reply:
I'd be fine with that, but we have deliberately also tried to follow the official SDW development setup instructions so as not to deviate in ways that could skew the outcome of the CI/tests. 

If the SD team are happy for the `sd-dev` instance to be Debian, I'll happily change that, but I wonder if you want to make that official in the docs.

Reply:
Hm, the docs are not really straightforward but https://developers.securedrop.org/en/latest/workstation_setup.html#download-configure-copy-to-dom0 links to https://developers.securedrop.org/en/latest/setup_development.html#qubes which says to use Debian 11.

Reply:
Oh hah! Ok, so this has changed since the last time I looked :O

Thanks. I will reset the VMs and recreate the sd-dev as Debian 11 then! I wonder if in fact on Qubes 4.2 I'm going to get Debian 12 out of the box though and should use that, there..

Reply:
Well, I replaced the sd-dev with a Debian 11 VM and installed podman on it, and `make clone` failed.

```
INFO:[2024-03-13-02:33:50:073713] Running: make clone
INFO:[2024-03-13-02:33:50:325156] Building RPM on sd-dev ...
INFO:[2024-03-13-02:33:50:325927] make: *** [Makefile:49: clone] Error 127
INFO:[2024-03-13-02:33:50:326001] Exception occurred during: make clone
```

Turns out it's because `make` is a hard requirement (for sd-dev VM), not just docker or podman - this is not in the workstation docs and probably should be.